code/modules/organs/organ.dm

```diff
@@ -538,3 +550,10 @@ var/global/list/ailment_reference_cache = list()
 /obj/item/organ/proc/is_internal()
 	return FALSE
 
+//Used to tell stumps from other organs. Stumps don't behave like regular organs, and require special handling.
+/obj/item/organ/proc/is_stump()
```

Comment:
That's actually a proc from /obj/item/organ/external that I brought down to /obj/item/organ. Needed it here to avoid doing extra type check + cast since a lot of proc work on /organ in general.

Honestly I'm not sure if doing a type check is faster than using a proc. But I'm assuming if it was there someone probably checked it before?

Reply:
Typechecks should generally be avoided. Class shouldn't be used as a categorization system.